src/mapml/layers/MapLayer.js

```diff
@@ -94,7 +94,7 @@ export var MapMLLayer = L.Layer.extend({
                 if (properties) {
                   var c = document.createElement('div');
                   c.insertAdjacentHTML('afterbegin', properties.innerHTML);
-                  geometry.bindPopup(c, {autoPan:false});
+                  geometry.bindPopup(c, {autoPan:false, closeButton: false});
```

Comment:
Why are close buttons not useful in feature's popups?

Reply:
I think @shepazu mentioned they might be problematic, or it might have been me misinterpreting something...

Reply:
It was suggested that it be removed, should it be added back?

Reply:
I think having a close button is fine, with the following caveats:
- the popup should not be a modal (that is, it shouldn't be a focus trap)
- if possible, we shouldn't force screen reader users to "close" each feature popup to move on to another feature

This seems like somewhat competing experiences between voice-control and screen-reader users, in exposing the right amount of detail with the fewest possible number of forced interactions. My preference would be to have the popup **not** exposed to screen readers on focus, but only read the accessible name (the popup "title"), with notification of additional information (e.g. `aria-expanded`) to access the other popup details only on demand (at which point they might encounter the close button, but that's fine, because they chose to expand it). For voice-control users, I think having the popup on focus should be good, with scrolling for long details, and a close button.

Does this seem right to you, @Malvoz?

Reply:
> I think having a close button is fine, with the following caveats:
>   - the popup should not be a modal (that is, it shouldn't be a focus trap)

The popups are (generally*) non-modal, so focus traps are mostly not an issue.

*when the popup for the last feature in the sequence is opened tabbing _is_ currently trapped to some extent (<kbd>shift</kbd> + <kbd>tab</kbd> can be used to move back to the previous feature):

<table>
<td width="250">
<video src="https://user-images.githubusercontent.com/26493779/107206251-1122c380-69ff-11eb-87ab-f30b5d33c7dd.mp4"></video>
</td>
</table>

> if possible, we shouldn't force screen reader users to "close" each feature popup to move on to another feature

Because focus (generally) isn't trapped inside popups users don't have to explicitly close popups before moving on to other features/focusable elements.

> This seems like somewhat competing experiences between voice-control and screen-reader users, in exposing the right amount of detail with the fewest possible number of forced interactions.

I'm not super familiar with voice-control so I'm not sure about how there are competing experiences here. I do know that some people use voice-control to move their cursor around to interact with elements, in that case I think a close button could be appreciated if they just wanted to close a popup without opening a new one.

> My preference would be to have the popup not exposed to screen readers on focus, but only read the accessible name (the popup "title"), with notification of additional information (e.g. `aria-expanded`) to access the other popup details only on demand (at which point they might encounter the close button, but that's fine, because they chose to expand it).

Yes I think we agreed that popups should not be opened (and thus exposed to ATs) when a feature recieves focus, and that each feature itself should be labelled for discernibility. (discussion from https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/270#issuecomment-770489852)

<!--
OTOH I think everything inside the popup should be announced once it's been opened, including the feature pagination buttons and the close button, and that's not because I personally think a screen reader user necessarily would want/need to have that information without navigating to those controls first, but because that would be the case if those controls existed inside a [`dialog` element](https://googlechrome.github.io/dialog-polyfill/), a screen reader would announce them. The good thing is that the close button currently resides after the author provided content in the DOM, so it'd be the last thing to be announced.-->

I agree that features that are interactive and have popups should expose the state of the popup using `aria-expanded`.